muspinsim_input/.shed.yml

```
@@ -0,0 +1,10 @@
+categories:
+- Computational Chemistry
+- Molecular Dynamics
```

Resuggesting this now I figured out how to comment on multiple lines 😅 
```suggestion
- Materials science
- Muon spectroscopy
```